testvectors/example.go

```diff
@@ -31,38 +26,31 @@ type:
   primitiveAssociative: true
 `)
 
-	var tests []*Vector
-	err := yaml.Unmarshal([]byte(`
-- name: list insert 1
-  schemaName: listordering-primitiveInlineList
-  lastObject: |
-      items:
-      - a
-      - b
-      - c
-  liveObject: |
-      items:
-      - a
-      - b
-      - c
-      - d
-  newObject: |
-      items:
-      - a
-      - b
-      - e
-      - c
-  expectedObject: |
-      items:
-      - a
-      - b
-      - e
-      - c
-      - d
-`), &tests)
-	if err != nil {
-		panic(fmt.Sprintf("test cases defined wrong: %v", err))
+	exampleSequence := &Sequence{
+		Name:       "example",
+		SchemaName: "listordering-primitiveInlineList",
+		InitialState: `items:
```

Comment:
If you look at it now, that I defined the actions inline it get's harder to read again.
So either we define the actions outside the sequence or (as you already said it would be preferred) get rid of the interface and make it yaml compatible again.
I still am not sure how to represent this in a good way and allow various different actions, though.